airflow/plural/docs/running-on-custom-nodes.md

```diff
@@ -0,0 +1,68 @@
+## Running Airflow Tasks on Custom Node Group
+
+In order to point your tasks at a custom node group, you will need to use the `KubernetesExecutor`
+
+There may be a desire to run your Airflow tasks on a specific node size for large workloads, or maybe even 
+[spot instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) to achieve higher cost 
+savings.
+
+> Disclaimer: if you run your Airflow workloads on spot instances, it is highly recommended to [set retries](https://docs.astronomer.io/learn/rerunning-dags) 
+> for your tasks as they may lose their underlying compute at any time
+
+### create custom node group
```

Comment:
We should add a mention that this step is optional only if you want to ensure at least 3 spot nodes are present at any time for latency